File: bob/bob.js
//  class Bob
//    def hey(input)
//    end
//  end

var Bob = function() {
  this.hey = function(input){
    if(input.toUpperCase() == input) {
      return "Woah, chill out!";
    } else if(input.substr(input.length-1) == "?") {
      return "Sure.";
    } else {
      return "Whatever.";
    };
  };
};

exports.Bob = Bob;

File: bob/bob.rb
class Bob

  def hey(input)
    if input.strip == ""
      "Fine. Be that way!"
    elsif input =~ /[a-zA-Z]/ && input.upcase == input
      "Woah, chill out!"
    elsif input.end_with?("?")
        "Sure."
    else
      "Whatever."
    end
  end

end
